[GTK] Build fix for Debian stable after 263331@main

dpino opened the pull request:
#### f407ecf491e1ee9b2fc63f594ef6413b0ceb8ec9
<pre>
[GTK] Build fix for Debian stable after 263331@main
<a href="https://bugs.webkit.org/show_bug.cgi?id=255518">https://bugs.webkit.org/show_bug.cgi?id=255518</a>

Reviewed by Adrian Perez de Castro.

Macro &apos;G_OPTION_ENTRY_NULL&apos; is only defined since glib 2.70, which is
not featured in Debian Stable.

* Tools/MiniBrowser/gtk/main.c: Replace macro &apos;G_OPTION_ENTRY_NULL&apos; for NULL.
(addSettingsGroupToContext):

Canonical link: <a href="https://commits.webkit.org/263707@main">https://commits.webkit.org/263707@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/b692d62f156c21838bcda50a5424a8980160ea11

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/5511 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/5657 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/5847 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/7056 "Built successfully") | [  ~~🛠 wincairo~~](https://ews-build.webkit.org/#/builders/32/builds/5520 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [❌ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/5509 "Failed to checkout and rebase branch from PR 13265") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/5882 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/5631 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/6878 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/5612 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/15/builds/5669 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/18/builds/4928 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/7090 "Built successfully") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/39/builds/3146 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/10/builds/4944 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/12086 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/5012 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/36/builds/5024 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/21/builds/6771 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/5462 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/17/builds/4475 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/4918 "Built successfully") | | | 
| | [  ~~🛠 watch~~](https://ews-build.webkit.org/#/builders/4/builds/9021 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/629 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/5280 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

aperezdc commented on Tools/MiniBrowser/gtk/main.c:
```diff
@@ -384,7 +384,7 @@ static gboolean addSettingsGroupToContext(GOptionContext *context, WebKitSetting
             .arg_description = "FEATURE-LIST",
         },
-        G_OPTION_ENTRY_NULL
+        NULL
```
Good catch, I see that `G_OPTION_ENTRY_NULL` exists since GLib 2.70 only. But this should be `{ NULL }` because otherwise the compiler may complain that a struct is being initialized with a scalar:

```suggestion
        { NULL }
```

aperezdc reviewed (approved):
Please apply the suggested change before landing, otherwise patch LGTM.

dpino committed:
[GTK] Build fix for Debian stable after 263331@main
https://bugs.webkit.org/show_bug.cgi?id=255518

Reviewed by Adrian Perez de Castro.

Macro 'G_OPTION_ENTRY_NULL' is only defined since glib 2.70, which is
not featured in Debian Stable.

* Tools/MiniBrowser/gtk/main.c: Replace macro 'G_OPTION_ENTRY_NULL' for NULL.
(addSettingsGroupToContext):

Canonical link: https://commits.webkit.org/263707@main
